nms/shared/sequelize_models/index.ts

```diff
@@ -47,50 +50,56 @@ const db = createNmsDb(sequelize);
 
 function createNmsDb(sequelize: Sequelize) {
   const db = {
-    AuditLogEntry: AuditLogEntryModel(sequelize, Sequelize),
-    FeatureFlag: FeatureFlagModel(sequelize, Sequelize),
-    Organization: OrganizationModel(sequelize, Sequelize),
-    User: UserModel(sequelize, Sequelize),
+    AuditLogEntry: createAuditLogEntryModel(sequelize),
+    FeatureFlag: createFeatureFlagModel(sequelize),
+    Organization: createOrganizationModel(sequelize),
+    User: createUserModel(sequelize),
   };
 
-  Object.keys(db).forEach(
+  (Object.keys(db) as Array<keyof typeof db>).forEach(
     modelName => db[modelName].associate != null && db[modelName].associate(db),
   );
   return db;
 }
 
-// $FlowIgnore Cannot define type for userModel
-async function isMigrationNeeded(userModel): Promise<boolean> {
+async function isMigrationNeeded(userModel: UserModelStatic): Promise<boolean> {
   try {
     const allUsers = await userModel.findAll();
+
     if (allUsers.length > 0) {
       console.warn('Users found in target DB. Migration may already have run');
-      return await false;
+      return false;
     }
-    return await true;
+    return true;
   } catch (e) {
+    const message = e instanceof Error ? e.message : 'unknown error';
     console.error(
       `Unable to run migration. Connection error to specified database: \n` +
         `------------------------\n` +
-        `${e} \n` +
+        `${message} \n` +
```

Comment:
There is a utility `getErrorMessage()` which could be used here instead of the manual logic.